src/flp/types/fixedpoint_l2.rs

```diff
@@ -792,52 +792,49 @@ mod tests {
         let mut input: Vec<Field128> = vsum.encode_measurement(&fp_vec).unwrap();
         assert_eq!(input[0], Field128::zero());
         input[0] = one; // it was zero
-        flp_validity_test(
-            &vsum,
-            &input,
-            &ValidityTestCase::<Field128> {
-                expect_valid: false,
-                expected_output: Some(vec![
-                    Field128::from(enc_vec[0] + 1), // = enc(0.25) + 2^0
-                    Field128::from(enc_vec[1]),
-                    Field128::from(enc_vec[2]),
-                ]),
-                num_shares: 3,
-            },
-        )
-        .unwrap();
+        FlpTest {
+            name: None,
+            flp: &vsum,
+            input: &input,
+            expected_output: Some(&vec![
```

Comment:
Clippy is flagging this as a useless `vec![]`. I think the same could be said about some of the expected output arguments passed to `expect_valid()`, but I'm not sure why Clippy didn't flag those. Maybe generic parameters interfere with the analysis?